app/main.py

```diff
@@ -1,8 +1,19 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = [Person(peo["name"], peo["age"]) for peo in people]
+    for hus in people:
+        if "wife" in hus and hus.get("wife"):
```

Comment:
You can simplify this: just remove "wife" in hus
The same with husband
```suggestion
        if hus.get("wife"):
```